app/models/course/assessment/answer/scribing_scribble.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+class Course::Assessment::Answer::ScribingScribble < ActiveRecord::Base
+  belongs_to :answer, class_name: Course::Assessment::Answer::Scribing.name, inverse_of: :scribbles
+  belongs_to :user, inverse_of: nil, foreign_key: true
```

Comment:
Would suggest that you use creator and updater columns instead, since we use it extensively. You can do that by modifying the migrations, and the other methods would be magically added in.